ui.frontend/webpack.common.js

```diff
@@ -90,6 +93,7 @@ module.exports = {
             }
         ]
     },
+    resolve: { alias },
```

Comment:
This should only be needed for dev builds when using `npm link` and when the linked folders have a `node_modules` folder. 

Production builds should use released dependencies only, and therefore the alias resolution is not required.